src/sentry/static/sentry/app/views/projectDebugFiles.jsx

```diff
@@ -0,0 +1,252 @@
+        <PanelItem key={key} align="center" px={2} py={1}>
+          <Box w={4.5 / 12}>
+            <code className="small">{dsym.debugId || dsym.uuid}</code>
+            <Flex mt="4px">
+              <Box w={4 / 12} pl="2px">
+                <p className="m-b-0 text-light small">
+                  <FileSize bytes={dsym.size} />
+                </p>
+              </Box>
+              <Box w={8 / 12} pl={1}>
+                <p className="m-b-0 text-light small">
+                  <span className="icon icon-clock" />{' '}
+                  <TimeSince date={dsym.dateCreated} />
+                </p>
+              </Box>
+            </Flex>
+          </Box>
+          <Box flex="1">
+            {dsym.symbolType === 'proguard' && dsym.objectName === 'proguard-mapping'
+              ? '-'
+              : dsym.objectName}
+            <p className="m-b-0 text-light small">
+              {dsym.symbolType === 'proguard' && dsym.cpuName === 'any'
+                ? 'proguard'
+                : `${dsym.cpuName} (${dsym.symbolType})`}
+            </p>
+          </Box>
+
+          <Box flex="1">
+            {access.has('project:write') ? (
+              <div className="btn-group pull-right">
+                <ActionLink
+                  onAction={() => (window.location = url)}
+                  className="btn btn-default btn-sm"
+                >
+                  <InlineSvg src="icon-download" /> {t('Download')}
+                </ActionLink>
+                <LinkWithConfirmation
+                  className="btn btn-danger btn-sm"
+                  title={t('Delete')}
+                  message={t(
+                    'Are you sure you wish to delete this debug infromation file?'
+                  )}
+                  onConfirm={() => this.onDelete(dsym.id)}
+                >
+                  <InlineSvg src="icon-trash" /> {t('Delete')}
+                </LinkWithConfirmation>
+              </div>
+            ) : null}
+          </Box>
+        </PanelItem>
+      );
+    });
+
+    return rows;
+  },
+
+  renderStreamBody() {
+    let body;
+
+    if (this.state.loading) body = this.renderLoading();
+    else if (this.state.error) body = <LoadingError onRetry={this.fetchData} />;
+    else if (this.state.debugFiles.length > 0) body = this.renderDsyms();
+    else if (this.state.query && this.state.query !== '')
+      body = this.renderNoQueryResults();
+    else body = this.renderEmpty();
+
+    return body;
+  },
+
+  render() {
+    return (
+      <div>
+        <SettingsPageHeader title={t('Debug Information Files')} />
+        <TextBlock>
+          {t(
+            `
+          Here you can find all your uploaded debug information files (dSYMs, ProGuard, Breakpad ...).
+          This is used to convert addresses and minified function names from crash dumps
+          into function names and locations.
+        `
+          )}
+        </TextBlock>
+
+        <div className="ref-project-releases">
```

Comment:
These class names should be updated (and are they even in use?) -- actually we don't even need all of these `div`s